src/Ui/UiRequest.py

```diff
@@ -276,7 +276,7 @@ def actionWrapper(self, path, extra_headers=None):
             if self.isWebSocketRequest():
                 return self.error403("WebSocket request not allowed to load wrapper")  # No websocket
 
-            if "text/html" not in self.env.get("HTTP_ACCEPT", ""):
+            if all([accept not in self.env.get("HTTP_ACCEPT", "") for accept in ("text/html", "text/*", "*/*")]):
```

Comment:
maybe rename `accept` to `mime_type` or something similar to make this line clearer in intent

Reply:
@skwerlman Done. Though @HelloZeroNet refused to merge, so I'm not sure if this is helpful.